colectare-date/colectare-date/Controllers/LoginController.cs

```diff
@@ -0,0 +1,32 @@
+﻿using Microsoft.AspNetCore.Mvc;
+
+namespace colectare_date.Controllers
+{
+    public class LoginController : Controller
+    {
+        [HttpGet]
+        public IActionResult Index()
+        {
+            return View();
+        }
+
+        [HttpPost]
+        public IActionResult Index(string email, string parola)
+        {
+            if (email == "admin@proiect.ro" && parola == "admin123")
```

Comment:
Haideti sa salvam aceste date in db, ca sa putem avea mai multi utilizatori si sa respectam practicile bune de securitate